app/src/features/projects/view/ProjectPage.tsx

```diff
@@ -75,25 +75,25 @@ const ProjectPage: React.FC = () => {
     }
   }, [urlParams, biohubApi, isLoadingCodes, codes]);
 
-  useEffect(() => {
-    const getProject = async () => {
-      const codesResponse = await biohubApi.codes.getAllCodeSets();
-      const projectWithDetailsResponse = await biohubApi.project.getProjectForView(urlParams['id']);
+  const getProject = async () => {
```

Comment:
No changes here, I just moved it out of the useEffect so that it can be passed down to the child components